docs/usage/ipv6.md

```diff
@@ -114,71 +102,58 @@ spec:
   ...
 ```
 
-### Explanation
-- **`spec.networking.ipFamilies`**: Specifies the stack (IPv4, IPv6, or both). In this example, both IPv4 and IPv6 are defined for a DualStack cluster.
-- **Annotation**: `provider.extensions.gardener.cloud/use-flow: "true"` is mandatory for DualStack support.
-
 ---
 
-## Migration of IPv4-only Shoot Clusters to Dual-Stack
+## Migration of IPv4-only Shoot Clusters to DualStack
 
-Eventually, migration should be as easy as changing the `.spec.networking.ipFamilies` field in the `Shoot` resource from `IPv4` to `IPv4, IPv6`.
-However, as of now, this is not supported.
-
-It is worth recognizing that the migration from an IPv4-only shoot cluster to a dual-stack shoot cluster involves rolling of the nodes/workload as well.
-Nodes will not get a new IPv6 address assigned automatically.
-The same is true for pods as well.
-Once the migration is supported, the detailed caveats will be documented here.
+To trigger a migration of an IPv4 shoot cluster to DualStack, both ipFamilies needs to be specified under `spec.networking.ipFamilies` in the shoot.
+Once the migration is triggered a constraint of type `ToDualStackMigration` is added to the shoot status. It is in progressing state.
+With the next shoot maintenance, the infrastructure is migrated to IPv6. The subnets get an external IPv6 range, the node subnet gets the secondary IPv4 range. The pod specific cloud routes are deleted from the VPC route table and alias IP ranges for the pod routes are added to the NIC of kubernetes nodes/instances. After that the status of the constraint will be changed to `DualStackInfraReady`. 
+With the next node roll-out, nodes will get IPv6 addresses and an IPv6 prefix for pods. When all nodes have IPv4 and IPv6 pod ranges, the status will be changed to `DualStackNodesReady`.
+The next reconcile will change all the remaining components to DualStack. Once the migration has finished the constraint will be removed with the next reconcile.
```

Comment:
Might need changes after updates of the migration PR in gardener/gardener, e.g. the name of the constraint might change.